docs/reference/modules/threadpool.asciidoc

```diff
@@ -180,7 +180,7 @@ the number of detected processors. This can be done by explicitly setting the
 `node.processors` setting. This setting is bounded by the number of available
 processors and accepts floating point numbers, which can be useful in environments
 where the {es} nodes are configured to run with CPU limits, such as cpu
-shares or quota under `Cgroups`. 
+shares or quota under `Cgroups`.
```

Comment:
This is a trailing whitespace that the IDE insisted on removing and I thought it's harmless to include it in this PR so I kept it.